www/src/data/sidebars/doc-links.yaml

```diff
@@ -460,6 +460,8 @@
           link: /docs/convincing-others/
         - title: Your First Professional Project
           link: /docs/first-professional-project/
+        - title: Setting Up Gatsby Without Gatsby New
```

Comment:
I think this is fine here, but we could add a link to it from the Preparing Your Environment page. (I don't think it fits very well in that section of the sidebar)

Reply:
I was a bit conflicted about it. It feels like an odd fit for most of the sections, but I have run into people who faced the same issue on Stack Overflow, so it seemed like this was an ok place.

Reply:
I went ahead and added a link from the Preparing Your Environment page, along with a link to the Quick start!